packages/internals/src/cli/getSchema.ts

```diff
@@ -63,7 +77,7 @@ async function readSchemaFromMultiFiles(schemaPath: string): Promise<GetSchemaRe
   debug('Ok')
 
   if (usesPrismaSchemaFolder(config)) {
-    return { schemaPath, schemas: files } as const
+    return { schemaPath, schemaRootDir: schemaPath, schemas: files } as const
```

Comment:
Why not `path.dirname(schemaPath)` like a few lines above?

Reply:
In case of directory, `schemaPath` points to directory, not a file, which is already "root directory of a schema"